insights/parsers/rabbitmq.py

```diff
@@ -146,18 +150,19 @@ def parse_content(self, content):
 class RabbitMQUsers(Parser):
 
     def parse_content(self, content):
-        users_dict = {}
+        self.data = {}
         for line in content[1:-1]:
-            line_splits = line.split()
+            line_splits = line.split(None, 1)
             if len(line_splits) > 1:
-                users_dict[line_splits[0]] = line_splits[1][1:-1]
-        self.data = users_dict
+                self.data[line_splits[0]] = line_splits[1][1:-1]
 
 
 @parser("rabbitmq_startup_log")
 class RabbitMQStartupLog(LogFileOutput):
     """Deprecated, use ``rabbitmq_log.RabbitMQStartupLog`` class instead."""
-    pass
+    def __init__(self, *args, **kwargs):
+        deprecated(RabbitMQStartupLog, 'Use the RabbitMQStartupLog class in the rabbitmq_log module')
+        super(self, RabbitMQStartupLog).__init__(*args, **kwargs)
```

Comment:
The arguments' order is reversed in the super().

Reply:
Oops, well spotted - thanks @xiangce.  Fixed.